### PR TITLE
feat: adds strings for the opt in toggles

### PR DESCRIPTION
**Pull request by jonybur**
This PR contains the strings for https://github.com/MetaMask/metamask-extension/pull/24605

## **Description**

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/24686?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -3202,6 +3214,36 @@
   "on": {
     "message": "On"
   },
+  "onboardedMetametricsAccept": {
```
Comment (NidhiKJha) [inline app/_locales/en/messages.json:3217]: Why do we need this? `onboardingMetametricsAgree` has the same value?

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -3202,6 +3214,36 @@
...
+    "message": "I agree"
+  },
+  "onboardedMetametricsDisagree": {
+    "message": "No thanks"
```
Comment (NidhiKJha) [inline app/_locales/en/messages.json:3221]: `onboardingMetametricsDisagree` has same value

**Review by NidhiKJha — COMMENTED** [review]
There is some repitition of values. Keys for these already exist in json

**Review by NidhiKJha — APPROVED** [review]

**Review by EtherWizard33 — APPROVED** [review]

**Comment by jonybur** [comment]
Can't be merged due to strings not being used

**Pull request closed without merging**